### ISSUE TITLE
MongoDB\BSON\UTCDateTime

### ISSUE THREAD

**Issue by acrozes** [alcaeus/mongo-php-adapter#73]
Just after update to php7 with doctrine mongoDb, have this error:
```
MongoResultException: MongoDB\BSON\UTCDateTime::__construct() expects parameter 1 to be integer, float given
```

The error stack:
```
\vendor\alcaeus\mongo-php-adapter\lib\Mongo\MongoDate.php:80
\vendor\alcaeus\mongo-php-adapter\lib\Alcaeus\MongoDbAdapter\TypeConverter.php:43
\vendor\alcaeus\mongo-php-adapter\lib\Alcaeus\MongoDbAdapter\TypeConverter.php:49
\vendor\alcaeus\mongo-php-adapter\lib\Alcaeus\MongoDbAdapter\TypeConverter.php:49
\vendor\alcaeus\mongo-php-adapter\lib\Mongo\MongoCollection.php:339
\vendor\doctrine\mongodb\lib\Doctrine\MongoDB\Collection.php:966
\vendor\doctrine\mongodb\lib\Doctrine\MongoDB\Collection.php:152
\vendor\doctrine\mongodb\lib\Doctrine\MongoDB\LoggableCollection.php:102
\vendor\doctrine\mongodb-odm\lib\Doctrine\ODM\MongoDB\Persisters\DocumentPersister.php:252
\vendor\doctrine\mongodb-odm\lib\Doctrine\ODM\MongoDB\UnitOfWork.php:1117
\vendor\doctrine\mongodb-odm\lib\Doctrine\ODM\MongoDB\UnitOfWork.php:425
\vendor\doctrine\mongodb-odm\lib\Doctrine\ODM\MongoDB\DocumentManager.php:526
\vendor\nelmio\alice\src\Nelmio\Alice\Persister\Doctrine.php:41
\src\AC\Tests\WebTestCase.php:44
\src\AC\Tests\MongoDbWebTestCase.php:24
```
Do you need more info ?

**Comment by acrozes** [comment]
Fix by pecl/mongodb 1.1.3

**Comment by alcaeus** [comment]
Thanks for the update, glad it's fixed with the new driver.
